lochness/rpms/__init__.py

```diff
@@ -63,7 +96,7 @@ def initialize_metadata(Lochness: 'Lochness object',
 
     source_source_name_dict = {
         'beiwe': 'Beiwe', 'xnat': 'XNAT', 'dropbox': 'Drpbox',
-        'box': 'Box', 'mediaflux': 'Mediaflux',
+        'box': 'Box',
```

Comment:
Sorry, I didn't notice it before--is the value spelt `Drpbox` with missing `o`?

Reply:
Nice catch - thanks for this. Fixed in a505f80.